static/data-ingestion.yaml

```
@@ -285,521 +305,95 @@ paths:
-    #                      #                          "parameters": [
-    #                      #                            {
-    #                      #                              "name": "id",
-    #                      #                              "value": "912"
-    #                      #                            }
-    #                      #                          ]
-    #                      #                        }
-    #                      #                      ],
-    #                      "modifiedAt": "2023-07-21 23:00:38"
-    #                    },
-    #                    {
-    #                      "sku": "es-product-2",
-    #                      "websiteCode": "us",
-    #                      "storeCode": "us",
-    #                      "storeViewCode": "us_en",
-    #                      "name": "es-product-2",
-    #                      "productId": 915,
-    #                      "type": "simple",
-    #                      "metaDescription": "es-product-2 ",
-    #                      "metaKeyword": "es-product-2",
-    #                      "metaTitle": "es-product-2",
-    #                      "status": "Enabled",
-    #                      "taxClassId": "Taxable Goods",
-    #                      "createdAt": "2023-07-21 14:20:44",
-    #                      "updatedAt": "2023-07-21 14:20:44",
-    #                      "urlKey": "es-product-2",
-    #                      "visibility": "Catalog, Search",
-    #                      "weightUnit": null,
-    #                      "currency": "USD",
-    #                      "buyable": true,
-    #                      "inStock": true,
-    #                      "lowStock": false,
-    #                      "parents": null,
-    #                      "url": "https://example.com/default/es-product-2.html",
-    #                      #                      "urlRewrites": [
-    #                      #                        {
-    #                      #                          "url": "https://example.com/es-product-2.html",
-    #                      #                          "parameters": [
-    #                      #                            {
-    #                      #                              "name": "id",
-    #                      #                              "value": "915"
-    #                      #                            }
-    #                      #                          ]
-    #                      #                        }
-    #                      #                      ],
-    #                      "modifiedAt": "2023-07-21 23:00:38"
-    #                    }
-    #                  ]
     patch:
       tags:
         - products
       summary: Product Feed API
-      description: >
-        Update specified products fields ony in Product Entity for given `sku` and `scopes`
+      description: Update Product entity for given `sku` and `scope`
```

```suggestion
      description: Update Product entity for given `sku` and `scope`. The "replace" strategy is used to update entity:   the entire field will be replaced with new value
```

The same we need to specify in other APIs for PATCH method

It would be good to provide an example where the product "name" and "roles" fields are updated and emphesase that for "roles" field whatever data was persisted before will be replaced with new values

---

Done

---

Done